src/components/UI/Title.css

```diff
@@ -0,0 +1,15 @@
+.banner{
+  display: flex;
+  align-items: center;
+}
+
+.header-title {
+  font-family: 'Noto Sans','Calibri', 'Arial', 'Sans Serif';
```

Comment:
shouldn't be necessary to assign font family. This should be inherited higher up.